.travis.yml

```diff
@@ -1,7 +1,8 @@
 language: scala
 sudo: true
 scala:
-- 2.11.8
+- 2.11.11
```

Comment:
Oh, but it should be possible to release for 2.12... I don't know travis well, but
https://github.com/joscha/play-easymail/blob/master/.travis.yml

it should run the "publish" for both versions, or ?